### ISSUE TITLE
Support PSID format in dump of configuration file and binding table

### ISSUE THREAD

**Issue by dpino** [Igalia/snabb#160]
Depends on #157

**Comment by andywingo** [comment]
From what I can tell, this feature isn't needed.  The reason is that right now the lwaftr doesn't have a northbound interface, so there has to be some other program configuring the lwaftr, or otherwise it's done by hand.  In that case you know the configuration -- it's the way you configured it!  If the lwaftr could change its configuration while running, then we would need this, but as is I think we might be able to punt on this feature entirely.  WDYT?  Am I missing something?

**Comment by kbara** [comment]
I originally had a similar series of thoughts. The missing factor is that it was explicitly requested. It's not useless, especially now that we can change configurations on the fly.

**Comment by andywingo** [comment]
We can reload configuration from the same file, sure.  But the configuration will always be in that file, and there should ideally be only one entity (human or robot) editing that file, so I dunno, it seems of limited utility.

**Comment by dpino** [comment]
The little code that landed addressing this issue (src/apps/lwaftr/dump.lua) uses the old conf and binding table formats. In other words, it's broken. I agree this feature is of little help at this moment. We can remove the current dump code (in `apps/lwaftr/dump.lua`) and leave a TODO in the handling of the signal.

```lua
elseif msg.kind == messages.lwaftr_message_dump_config then
-- TODO: Dump configuration and binding table to a file.
```

**Comment by kbara** [comment]
It's an extremely useful feature, in practice. For long-running processes, in particular, it may well not correspond with what's on disk if someone changes it without reloading the config. Let's put it back this week, though not tonight. It *should* work in 2.1.

**Comment by dpino** [comment]
Solved in #297.
